src/components/Reel/Reel.styled.js

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+
+const ReelBody = styled.section`
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: space-between;
+	width: min(100% - 2rem);
+	height: 4em;
+	margin: 0 auto;
+	margin-bottom: 1em;
+	overflow: hidden;
+	overflow-x: scroll;
+	border-radius: 1em;
+	background: transparent;
+	scrollbar-width: 0;
+
+	article {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		width: 4em;
+		height: 4em;
+		margin-left: 0.7em;
+		padding: 1em;
+		border-radius: 1em;
+		background: transparent;
+		box-shadow: inset 0 0 50px 5px #00dfc4;
+		size: large;
+		color: #00dfc4;
+	}
+
+	article:nth-child(1) {
+		transition: 1s;
+		box-shadow: ${props =>
+			props.active === 'all' ? 'inset 0 0 15px 3px #FF5F1F' : 'inset 0 0 15px 3px #00dfc4'};
+		color: ${props => (props.active === 'all' ? '#FF5F1F' : '')};
```

Comment:
There should be no case in which color is empty.

Reply:
Done